keyboards/hhkb/ansi/keymaps/naut/keymap.c

```diff
@@ -0,0 +1,23 @@
+/*  -*-  eval: (turn-on-orgtbl); -*-
+ * naut's HHKB Layout
+ */
+#include QMK_KEYBOARD_H
+
+#define BASE 0
+#define HHKB 1
```

Comment:
We recommend enums over defines for layer naming, since it's easier to maintain over time, and less prone to empty layers artificially bloating the firmware size.

```suggestion
enum custom_layers {
    BASE,
    HHKB,
};
```

Reply:
GPL2+ compatible license header added and layer naming changed to enums.